src/main/java/org/ngafid/ProcessUpload.java

```diff
@@ -298,25 +302,25 @@ public static boolean ingestFlights(Connection connection, int uploadId, int fle
         HashMap<String, UploadException> flightErrors = new HashMap<String, UploadException>();
 
         int validFlights = 0;
-        int warningFlights = 0;
-        int errorFlights = 0;
+        AtomicInteger warningFlights = new AtomicInteger();
```

Comment:
I don't think these need to be atomic since we are not using a parallel stream. Though, we may want to use 'forEachOrdered' instead of `forEach` to make this more explicit.